airline-web/app/controllers/Application.scala

```diff
@@ -509,6 +509,39 @@ class Application @Inject()(cc: ControllerComponents, val configuration: play.ap
     }))
   }
 
+  def getLinksByAirplaneModel(modelId : Int) = Action {
+    val linksByModel = LinkSource.loadFlightLinksByAirplaneModel(airportId).groupBy(_.from)
```

Comment:
@Bohaska the airportId here is not defined. Meanwhile the modelId is not being used.

Reply:
I reused some code from the airport flight map function and forgot to change some of the parameters...

Yea, swap airportId with modelId